src/main/resources/application.properties

```diff
@@ -32,3 +32,6 @@ sge.execd.port=6445
 sge.qmaster.host.path=/opt/sge/default/common/act_qmaster
 sge.parallel.environment.registration.default.slots=${GRID_ENGINE_API_PE_REGISTRATION_DEFAULT_SLOTS:999}
 sge.parallel.environment.registration.default.allocation.rule=${GRID_ENGINE_API_PE_REG_DEFAULT_ALLOCATION_RULE:$fill_up}
+
+#SLUEM specific properties
+slurm.job.output-fields-count=52
```

Comment:
Please, use current approach to properties configuration - configurable via environment variable with default value set to `52`

Reply:
fixed